tests/backends/test_cursors.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from django.core.exceptions import ImproperlyConfigured
+from django.db import connection
+from django.test import TestCase
+
+
+class LoggingCursorMixin:
+    def __init__(self, *args, **kwargs):
+        bucket = kwargs.pop('bucket')
+        super().__init__(*args, **kwargs)
+        self.bucket = bucket
+
+    def execute(self, sql, *args, **kwargs):
+        self.bucket.append(sql)
+        super().execute(sql, *args, **kwargs)
+
+
+@unittest.skipUnless(connection.vendor == 'mysql', 'MySQL specific test.')
+class MySQLCursorOptionsTestCase(TestCase):
+    try:
```

Comment:
This check is redundant. `skipUnless` already guarantees that we have MySQL. You can remove the `try ... except` leaving only `import`.